frontend/src/scenes/funnels/People.tsx

```diff
@@ -9,9 +9,8 @@ import './FunnelPeople.scss'
 import { Card } from 'antd'
 
 export function People(): JSX.Element | null {
-    const { stepsWithCount, peopleSorted, peopleLoading, areFiltersValid } = useValues(funnelLogic({}))
-
-    if (!stepsWithCount || !areFiltersValid) {
```

Comment:
removed `areFiltersValid` because that just checks there's a minimum of 2 steps, but we still want to show this table even for just 1 step in the funnel

Reply:
Thanks for the context!